easybuild/easyblocks/g/gromacs.py

```diff
@@ -468,18 +468,29 @@ def sanity_check_step(self):
             libnames.extend([libname + mpisuff for libname in libnames])
 
         suff = ''
-        # add the _d suffix to the suffix, in case of the double precission
-        if re.search('DGMX_DOUBLE=(ON|YES|TRUE|Y|[1-9])', self.cfg['configopts'], re.I):
-            suff = '_d'
 
-        libs = ['lib%s%s.%s' % (libname, suff, self.libext) for libname in libnames]
+        # make sure that configopts is a list:
+        configopts = self.cfg['configopts']
+        if type(configopts) is str:
+            configopts = [configopts]
+
+        libs = []
+        binaries = []
+
+        for cfgopts in configopts:
+            # add the _d suffix to the suffix, in case of the double precission
+            if re.search('DGMX_DOUBLE=(ON|YES|TRUE|Y|[1-9])', cfgopts, re.I):
+                suff = '_d'
+
+            libs += ['lib%s%s.%s' % (libname, suff, self.libext) for libname in libnames]
+            binaries += [b + suff for b in bins]
```

Comment:
please use `.extend` rather than `+=` (since that avoid creating a new list on every iteration):

```python
libs.extend([...])
binaries.extend([...])
```

Maybe also rename `binaries` to `bin_files` (and `libs` to `lib_files` for consistency)?